### PR TITLE
Migrated TC test_profile_operations

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_profile_operations](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_profile_operations.py) and merged TC [test_profile_info_without_having_profile_started](https://github.com/gluster/glusto-tests/blob/master/tests/functional/glusterd/test_profile_info_without_having_profile_started.py) into it.

Fixes: #486 and Fixes: #487 
Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

File: common/ops/gluster_ops/volume_ops.py [hunk 1]
```diff
@@ -52,19 +56,19 @@ def setup_volume(self, volname: str, node: str, conf_hash: dict,
 
         # Create volume
         ret = self.volume_create(volname, node, conf_hash, server_list,
-                                 brick_root, force)
+                                 brick_root, force, excep)
```
Comment (srijan-sivakumar) [inline common/ops/gluster_ops/volume_ops.py:59]: Lets keep the excep for volume create as optional with default being True..

**Comment by nik-redhat** [comment]
Test Case output:
```
Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   0   |      0       |
|   Disruptive  |   1   |    100.0     |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 test_profile_operations
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     dist    |     PASS    | 48.21655988693237  |
|     rep     |     PASS    | 48.14690589904785  |
|   dist-rep  |     PASS    | 64.23775243759155  |
|     disp    |     PASS    | 48.197420597076416 |
|  dist-disp  |     PASS    | 61.43552541732788  |
+-------------+-------------+--------------------+



Framework runtime : 273.1510033607483
```

File: common/ops/gluster_ops/volume_ops.py (hunk 1 above)
Comment (nik-redhat) [reply common/ops/gluster_ops/volume_ops.py:59]: The default is True, only.

Comment (nik-redhat) [reply common/ops/gluster_ops/volume_ops.py:59]: I think you might miss the notification for this, so commenting again :p

Comment (srijan-sivakumar) [reply common/ops/gluster_ops/volume_ops.py:59]: Gosh..right!! I missed this..Should go to sleep now.. :p

Will review tomorrow.

**Review by srijan-sivakumar — APPROVED** [review]
LGTM

**Pull request merged**